thirdparty/icu4c/LICENSE

```diff
@@ -2,7 +2,7 @@ UNICODE LICENSE V3
 
 COPYRIGHT AND PERMISSION NOTICE
 
-Copyright © 2016-2023 Unicode, Inc.
+Copyright © 2016-2024 Unicode, Inc.
```

Comment:
Not sure why we have `Copyright: 1991-2021, Unicode` in `COPYRIGHT.txt` but it should likely be replaced by this one (`Copyright: 2016-2024, Unicode, Inc.`).

Reply:
Updated it.